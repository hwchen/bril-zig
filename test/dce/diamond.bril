../../bril/examples/test/tdce/diamond.bril