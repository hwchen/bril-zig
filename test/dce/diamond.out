@main {
  a: int = const 47;
  cond: bool = const true;
  br cond .left .right;
.left:
  jmp .end;
.right:
  jmp .end;
.end:
  print a;
}
